Gems/PhysX/Code/Source/StaticRigidBodyComponent.cpp

```diff
@@ -197,4 +226,157 @@ namespace PhysX
         return AzPhysics::SceneQueryHit();
     }
 
+    AZ::Vector3 StaticRigidBodyComponent::GetCenterOfMassWorld() const
+    {
+        return {};
```

Comment:
Math types don't default initialize, so you probably need to return CreateZero, CreateIdentity etc for these as appropriate.

Reply:
Well spotted! I'll change that, thanks!